src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralServiceTest.kt

```diff
@@ -534,265 +543,94 @@ class ReferralServiceTest @Autowired constructor(
-      val result = referralService.getSentReferralsForUser(user, null, null, false, null, null)
-      assertThat(result).containsExactlyInAnyOrder(selfAssignedReferral, otherAssignedReferral)
-      assertThat(result).doesNotContain(draftReferral, completedReferral, liveReferral, cancelledReferral)
-    }
-
-    @Test
-    fun `setting assigned to returns referrals with correct assignments`() {
-      val result = referralService.getSentReferralsForUser(user, null, null, false, otherUser.id, null)
-      assertThat(result).containsExactlyInAnyOrder(otherAssignedReferral)
-      assertThat(result).doesNotContain(draftReferral, completedReferral, liveReferral, cancelledReferral, selfAssignedReferral)
-    }
-
-    @Test
-    fun `setting assigned to with unknown user returns no referrals`() {
-      val result = referralService.getSentReferralsForUser(user, null, null, false, "unknown", null)
-      assertThat(result).isEmpty()
-      assertThat(result).doesNotContain(draftReferral, completedReferral, liveReferral, cancelledReferral, otherAssignedReferral, selfAssignedReferral)
-    }
-
-    @Test
-    fun `setting all filter options to true will return cancelled and unassigned referral`() {
-      val result = referralService.getSentReferralsForUser(user, true, true, true, null, null)
-      assertThat(result).containsExactlyInAnyOrder(cancelledReferral)
-      assertThat(result).doesNotContain(draftReferral, completedReferral, liveReferral, selfAssignedReferral, otherAssignedReferral)
-    }
-
-    @Test
-    fun `setting all filter options to false will return an assigned referral`() {
-      val result = referralService.getSentReferralsForUser(user, false, false, false, user.id, null)
-      assertThat(result).containsExactlyInAnyOrder(selfAssignedReferral)
-      assertThat(result).doesNotContain(draftReferral, completedReferral, liveReferral, cancelledReferral, otherAssignedReferral)
-    }
-  }
-
   @Nested
   @DisplayName("get sent referrals with a provider user")
   inner class GetSentReferralsSPUser {
-    private lateinit var otherPrime: ServiceProvider
-    private lateinit var otherSub: ServiceProvider
-
-    private fun newReferralWithProviders(prime: ServiceProvider, vararg subs: ServiceProvider): Referral {
-      val intervention = interventionFactory.create(
-        contract = contractFactory.create(primeProvider = prime, subcontractorProviders = subs.toSet())
-      )
-      return referralFactory.createSent(intervention = intervention)
-    }
-
-    @BeforeEach
-    fun setup() {
-      val anySpec: Specification<Referral> = any()
-      whenever(referralAccessFilter.serviceProviderReferrals(anySpec, any()))
-        .then(AdditionalAnswers.returnsFirstArg<Specification<Referral>>())
-      otherPrime = serviceProviderFactory.create("other_prime")
-      otherSub = serviceProviderFactory.create("other_sub")
-    }
-
-    @Test
-    fun `user with multiple providers can see referrals where the providers are subcontractors`() {
```

Comment:
this test isn't valid because we don't do anything with providers in the code path under test here anymore.